iphone/Classes/TiUIiPhoneAlertDialogStyleProxy.m

```diff
@@ -16,10 +16,10 @@ -(NSString*)apiName
     return @"Ti.UI.iPhone.AlertDialogStyle";
 }
 
-MAKE_SYSTEM_PROP(DEFAULT,                   UIAlertViewStyleDefault);
-MAKE_SYSTEM_PROP(PLAIN_TEXT_INPUT,          UIAlertViewStylePlainTextInput);
-MAKE_SYSTEM_PROP(SECURE_TEXT_INPUT,         UIAlertViewStyleSecureTextInput);
-MAKE_SYSTEM_PROP(LOGIN_AND_PASSWORD_INPUT,  UIAlertViewStyleLoginAndPasswordInput);
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(DEFAULT,                   UIAlertViewStyleDefault,       @"UI.iPhone.AlertDialogStyle.DEFAULT",@"6.0.0",@"UI.iOS.AlertDialogStyle.DEFAULT");
```

Comment:
Indentation looks weird, better indent it with normal space as done in newer API's.